app/modules/enduser_setup.c

```diff
@@ -1525,4 +1538,4 @@ static const LUA_REG_TYPE enduser_setup_map[] = {
   { LNILKEY, LNILVAL}
 };
 
-NODEMCU_MODULE(ENDUSER_SETUP, "enduser_setup", enduser_setup_map, NULL);
+NODEMCU_MODULE(ENDUSER_SETUP, "enduser_setup", enduser_setup_map, NULL);
```

Comment:
GitHub indicates that @FrankX0 replaced a line ending with newline by one without newline - and it doesn't like that (neither do I).